model/host/host.go

```diff
@@ -739,3 +744,52 @@ func CountInactiveHostsByProvider() ([]InactiveHostCounts, error) {
 	}
 	return counts, nil
 }
+
+// find all containers
+func FindAllContainers() ([]Host, error) {
+	query := db.Query(bson.M{
+		ContainerIDKey: bson.M{"$exists": true},
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find all containers belonging to parent
+func (h *Host) GetContainers() ([]Host, error) {
+	if !h.IsParent {
+		return nil, errors.New("Host is not a parent of any containers")
+	}
+	query := db.Query(bson.M{
+		ParentIDKey: h.ParentID,
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find parent of container
+func (h *Host) GetParent() (*Host, error) {
+	if h.ParentID == "" {
+		return nil, errors.New("Host does not have a parent")
+	}
+
+	host, err := FindOneId(h.ParentID)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding parent")
+	}
+	if host == nil {
+		return nil, errors.New("Error finding parent")
```

Comment:
Nit: There was no "Error finding parent." Instead, "Parent not found" or similar.

Reply:
done